slapp/data_selection/utils.py

```diff
@@ -0,0 +1,43 @@
+import pathlib
+
+candidate_movie_names = ['concat_31Hz_0.h5', 'motion_corrected_video.h5']
```

Comment:
Why is this global?

Reply:
Because I use `slapp.utils.candidate_movie_names` in the test to create the appropriate files (or not) that this search function is hard-coded to look for,